TestPlaybooks/playbook-MISP_V2-Test.yml

```diff
@@ -625,25 +695,29 @@ tasks:
         simple: ${MISP.Event.ID}
       hash:
         simple: ${File.MD5}
+    continueonerror: true
```

Comment:
why?

Reply:
it was failing in my server, But I solve it, I believed that if its deprecated we can pass it if it failing, because we don't support it.
Any how solved it.

Reply:
if the command is deprecated then either remove it from the playbook or don't ignore the error.
no reason to run something without looking at the result.

if it works, we can leave it as is.